### ISSUE TITLE
An Example link isn't in text/markdown area.

### ISSUE THREAD

**Issue by imba-tjd** [dotnet/docs#8749]
see the example in `Notes to Callers` section.
source is on https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Enum.xml#L1063

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e2aeb1cd-e9be-f47e-e3aa-157da10cc725
* Version Independent ID: b76b5d5b-42c1-178d-5b19-4c49c27bb3e2
* Content: [Enum.IsDefined(Type, Object) Method (System)](https://docs.microsoft.com/en-us/dotnet/api/system.enum.isdefined?view=netcore-2.1#notes-to-callers)
* Content Source: [xml/System/Enum.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Enum.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by rpetrusha** [comment]
I don't think that I understand the issue here, @imba-tjd. The code reference is in an ECMAXML block; that's by design, since the Notes to Callers section isn't intended to be in markdown, And the code example is rendering successfully.

**Comment by imba-tjd** [comment]
@rpetrusha  Sorry. My original issue is this link itself was translated. See https://docs.microsoft.com/zh-cn/dotnet/api/system.enum.isdefined?view=netcore-2.1#%E8%B0%83%E7%94%A8%E6%96%B9%E8%AF%B4%E6%98%8E. Since this issue exists in all local repos, I suppose this is a source issue.
